src/java/us/kbase/templates/module_dockerfile.vm.properties

```diff
@@ -3,7 +3,6 @@ FROM kbase/sdkbase2:python
 #else
 FROM kbase/sdkbase2:latest
 #end
-MAINTAINER #if($username)${username}#{else}KBase Developer#{end}
```

Comment:
This tag has been deprecated

Reply:
We should replace it with LABEL maintainer="...."

Reply:
Done.

Reply:
That's interesting, review comments are added to the comment thread based on the day they're made (although the date still shows `now`) rather than being added to the end of the comment thread as one would expect

Reply:
Docker docs on MAINTAINER tag: https://docs.docker.com/engine/reference/builder/#maintainer-deprecated

Reply:
The PR is correct, the MAINTAINER directive has been obsolete for a while and newer image should use LABEL MAINTAINER=